site/content/docs/5.3/components/orange-navbar.md

```diff
@@ -202,7 +202,7 @@ An additional navbar (with text or icon items) can be added on the right of the
           </li>
           <li class="nav-item">
             <a href="#" class="nav-link nav-icon">
-              <img src="/docs/{{<param docs_version>}}/assets/img/navbar-contact.png" width="25" height="25" role="img" alt="User" loading="lazy">
```

Comment:
Wouldn't be worth mentioning in the migration guide as it is a change in the component? (slight change, but could be improved on users' side as well).